packages/Collapsible/stories/Collapsible.stories.mdx

```diff
@@ -0,0 +1,15 @@
+import { Meta, Story } from "@storybook/addon-docs/blocks";
+import { Props, Description } from "@storybook/addon-docs/blocks";
+import { Gap } from "storybook/assets/styles/common.styles";
+import Readme from "../README.md";
+import Collapsible from "../src";
+
+<Meta title=" | Collapsible/ Props" component={Collapsible} parameters={{ options: { showPanel: false } }} />
```

Comment:
I thought we would call this `Docs`, and probably have it as the first or second page?